### DISCUSSION TITLE
When running `yarn run devserver-hot` gives error: `AttributeError: module 'le_utils.constants.format_presets' has no attribute 'IMSCP_ZIP'`

### DISCUSSION THREAD

**Discussion by muditchoudhary** [learningequality/kolibri#11472] [Q&A] [answered]
After syncing my local `develop` branch with the upstream `develop` branch and then trying to run the dev server I got an error that

`AttributeError: module 'le_utils.constants.format_presets' has no attribute 'IMSCP_ZIP'`

Full logs are here: https://gist.github.com/muditchoudhary/9356b3d817ef8bac99e5e5b296bbdbe2

## Solution
For me upgrading the Python dependencies worked.
I typed all the **required** commands that are given on the _Install Python Dependency_ section

Docs link for the commands: https://kolibri-dev.readthedocs.io/en/develop/getting_started.html#install-python-dependencies

Os: Ubuntu 22.04.3 LTS
Branch: Develop the last sync till this commit: https://github.com/learningequality/kolibri/commit/e001ec0616f7be493db51af4e08ab4c329e11339

**Comment by rtibbles** [comment, marked as answer]
Yes, this is a result of the le utils dependency being out of date in your local environment, so reinstalling the dependencies will fix it.
